code/LeftAndMain.php

```diff
@@ -901,16 +901,12 @@ public function MenuCurrentItem()
     }
 
     /**
-     * Return appropriate template(s) for this class, with the given suffix using
+     * Return appropriate template candidates for this class, with the given suffix using
      * {@link SSViewer::get_templates_by_class()}
-     *
-     * @param string $suffix
-     * @return string|array
      */
-    public function getTemplatesWithSuffix($suffix)
+    public function getTemplatesWithSuffix(string $suffix): array
     {
-        $templates = SSViewer::get_templates_by_class(get_class($this), $suffix, __CLASS__);
-        return SSViewer::chooseTemplate($templates);
+        return SSViewer::get_templates_by_class(get_class($this), $suffix, __CLASS__);
     }
```

Comment:
This used to give specific absolute file paths to templates.
We can't do that anymore, because a) some template engines might not support that, and b) we don't want to pass a .ss template path to a twig template engine for example.

Thankfully, everywhere that used this doesn't actually _need_ full absolute file paths. They just need some template candidates, and the ability to ask the configured engine if it can find files from those candidates or not.